theme_aviato/__manifest__.py

```diff
@@ -56,14 +61,18 @@
         'static/description/aviato_screenshot.jpg',
     ],
     'images_preview_theme': {
-        'website.s_cover_default_image': '/theme_aviato/static/src/img/content/s_cover.jpg',
+        'website.s_image_text_default_image': '/theme_aviato/static/src/img/content/s_banner_2.jpg',
+        'website.s_product_list_default_image_1': '/theme_aviato/static/src/img/content/s_banner_3.jpg',
+        'website.s_picture_default_image': '/theme_aviato/static/src/img/content/s_popup.jpg',
+        'website.s_media_list_default_image_1': '/theme_aviato/static/src/img/content/s_wall_01.jpg',
```

Comment:
![image](https://github.com/user-attachments/assets/66cd0eb1-5b6c-4ca8-8cbd-b5fc0b6057c9)
There are 3 times the same image on `s_media_list`.
It was already like this before this PR, but could you fix it please